image_view/include/image_view/disparity_view_node.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef IMAGE_VIEW__DISPARITY_VIEW_NODE_HPP_
+#define IMAGE_VIEW__DISPARITY_VIEW_NODE_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+#include <stereo_msgs/msg/disparity_image.hpp>
+
+#include <opencv2/highgui/highgui.hpp>
+
+#include <memory>
+#include <string>
+
+namespace image_view
+{
+
+class DisparityViewNode
+  : public rclcpp::Node
+{
+  // colormap for disparities, RGB order
+  static unsigned char colormap[];
+
+  std::string window_name_;
+  rclcpp::Subscription<stereo_msgs::msg::DisparityImage>::SharedPtr sub_;
+  cv::Mat_<cv::Vec3b> disparity_color_;
+  bool initialized;
+
+  void imageCb(const stereo_msgs::msg::DisparityImage::SharedPtr msg);
+
+public:
```

Comment:
Just out of convention, can you place the constructor and destructor first in the headers?

Reply:
Done.